lib/samples/download_vector_tiles_to_local_cache/download_vector_tiles_to_local_cache.dart

```diff
@@ -211,17 +214,21 @@ class _DownloadVectorTilesToLocalCacheState
 
     _exportVectorTilesJob?.onStatusChanged.listen(
       (status) {
-        if (status == JobStatus.succeeded) {
-          setState(() => _progress = 100.0);
-          if (mounted) {
-            ScaffoldMessenger.of(context).showSnackBar(
-              SnackBar(
-                content: const Text('Downloaded vector tiles successfully'),
-                duration: const Duration(seconds: 5),
-                backgroundColor: Theme.of(context).colorScheme.secondary,
-              ),
-            );
-          }
+        if (status == JobStatus.succeeded && mounted) {
+          ScaffoldMessenger.of(context).showSnackBar(
+            SnackBar(
+              content: const Text('Downloaded vector tiles successfully'),
+              duration: const Duration(seconds: 5),
+              backgroundColor: Theme.of(context).colorScheme.secondary,
+            ),
+          );
+
+          // delay 5 seconds to dismiss the progress indicator.
```

Comment:
```suggestion
          // Hide the progress indicator after a 5 second delay so that the user can see the job completed.
```
Since this adds more boilerplate I think we need to add more of an explanation.